demo_plugin/newhelm/benchmarks/demo_benchmark.py

```diff
@@ -17,13 +17,13 @@
 class DemoBenchmark(BaseBenchmark):
     """A benchmark that runs all of the Demo Tests."""
 
-    def get_tests(self) -> List[BaseTest]:
-        """Returns the list of all Test objects this benchmark wants to run."""
-        return [
-            DemoSimpleQATest(),
-            DemoUnpackingDependencyTest(),
-            DemoPairedPromptsTest(),
-        ]
+    def get_tests(self) -> Mapping[str, BaseTest]:
```

Comment:
Could you say more about why you made the name external rather than part of the object?

Reply:
I've gone back and forth on this point, and I'm not strongly attached to external vs internal. The reason it currently is external is:

* Name, at least how its primarily used here, is context dependent. A Benchmark that uses one BBQ instance will just call it `BBQ` while a Benchmark using more than one BBQ will want to name each uniquely. The goal is to allow `summarize` to find the Results for each Test created in this step, and for the Records to make sense in the context of the Benchmark.
* The Test itself doesn't care that it has a name and doesn't need it to function. We absolutely don't want Test authors using name to control behavior.
* There are many objects which fit both these properties (SUTs, Measurements, Annotators).

Reply:
I see your point, but generally if two lumps of data end up moving around together, it's a sign to me that they're both part of the same object. I think the function of "which test is this exactly" is pretty closely bound up with the Test. I'm fine with this getting merged for now, but long term I'm suspicious of it.